spec/google/gax/util_pb.rb

```diff
@@ -0,0 +1,54 @@
+# Copyright 2017, Google Inc.
```

Comment:
Is this generated by `protoc` or handwritten? If it's generated, let's check in the proto file as well ([like Python](https://github.com/googleapis/gax-python/tree/master/tests/fixtures)), which makes it easier to modify in the future. Otherwise, let's replace this with a generated file.